packages/web/src/migrations/update-11-5-2/create-babelrc-for-workspace-libs.ts

```diff
@@ -0,0 +1,29 @@
+import { formatFiles, getProjects, Tree } from '@nrwl/devkit';
+import { reverse } from '@nrwl/workspace/src/core/project-graph';
+import { createProjectGraphFromTree } from '@nrwl/workspace/src/utilities/create-project-graph-from-tree';
+import { hasDependentAppUsingWebBuild } from '@nrwl/web/src/migrations/update-11-5-2/utils';
+
+export async function createBabelrcForWorkspaceLibs(host: Tree) {
```

Comment:
We can't perform this migration in our monorepo, I am going to investigate what is problem
![image](https://user-images.githubusercontent.com/5691926/111985446-e8691000-8b0c-11eb-9dd4-063b259eac1a.png)